### ISSUE TITLE
Crash in testsolv at testcase.c:2101

### ISSUE THREAD

**Issue by irfanariq** [openSUSE/libsolv#473]
Hello,

We are currently working on fuzz testing feature, and we found a crash on `testsolv`.

The stack traces are as follow:
```st
==612==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f85bea48e1a bp 0x7ffeeab6b230 sp 0x7ffeeab6a990 T0)
==612==The signal is caused by a READ memory access.
==612==Hint: address points to the zero page.
    #0 0x7f85bea48e19  (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x59e19)
    #1 0x560474cff5a1 in testcase_read .../libsolv-git/ext/testcase.c:2101
    #2 0x560474ced75b in main .../libsolv-git/tools/testsolv.c:159
    #3 0x7f85be402bf6 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x21bf6)
    #4 0x560474cec699 in _start (.../libsolv-git/install_asan/bin/testsolv+0x23699)
```
Here is the full stack trace : 
[stacktrace_testsolv_1.zip](https://github.com/openSUSE/libsolv/files/7078354/stacktrace_testsolv_1.zip)

**Step to reproduce**

We configured `testsolv` using `CFLAGS="-g -O0 -fsanitize=address" cmake -DDISABLE_SHARED=ON ..` and built it using `make -j10; make install`, and run it with:

```
./testsolv <attached file>
```
Attachment: 
[input_testsolv_1.zip](https://github.com/openSUSE/libsolv/files/7078356/input_testsolv_1.zip)

**Environment**
- OS: Ubuntu 18.04.5 LTS
- GCC version: gcc 7.5.0
- libsolv version: latest commit of master branch on github ([link](https://github.com/openSUSE/libsolv/tree/3d88b0ca5188e8d4a180ff30e7c183f71a367d0c))

Thank you.

**Comment by mlschroe** [comment]
Yeah, the testcase parser is not as robust as it should be. It's not used for dependency solving (i.e. dnf or zypper), so this is not a big issue. Still, I'm happy to fix all such bugs.

Fixed in commit 9d53fe7416fc546ade29aea50a51a43f4bfe0a94. Thanks for the report!

**Comment by irfanariq** [comment]
Thank you for confirming and fixing it
